test/kino/text_test.exs

```diff
@@ -0,0 +1,23 @@
+defmodule Kino.TextTest do
+  use Kino.LivebookCase, async: true
+
+  describe "new/1" do
+    test "outputs plain text" do
+      "Hello!" |> Kino.Text.new() |> Kino.render()
+      assert_output({:plain_text, "Hello!"})
+
+      "Hello!" |> Kino.Text.new(console: false) |> Kino.render()
+      assert_output({:plain_text, "Hello!"})
+    end
+
+    test "outputs console text" do
+      [:red, "Hello!"]
+      |> IO.ANSI.format()
+      |> IO.iodata_to_binary()
+      |> Kino.Text.new(console: true)
+      |> Kino.render()
+
+      assert_output({:text, "\e[31mHello!\e[0m"})
```

Comment:
On CI Elixir disables ANSI. We could enable it explicitly, but I don't think it's relevant for the test, the content will be whatever we pass, just need to make sure it's the right output type :)

```suggestion
      "Hello!" |> Kino.Text.new(console: true) |> Kino.render()
      assert_output({:text, "Hello!"})
```